C#/Botkit/Microsoft.BotKit/BotWorker.cs

```diff
@@ -36,30 +44,27 @@ public Botkit GetController()
         }
 
         /// <summary>
-        /// Get a value from the BotWorker's configuration.
-        /// </summary>
-        /// <param name="key">The name of a value stored in the configuration</param>
-        /// <returns>The value stored in the configuration (or null if absent)</returns>
-        public object GetConfig(string key)
-        {
-            return new object();
-        }
-
         public Activity GetActivity()
         {
             return new Activity();
         }
 
-        /// <summary>
         /// Send a message using whatever context the bot was spawned in or set using changeContext() --
         /// or more likely, one of the platform-specific helpers like startPrivateConversation() (Slack), startConversationWithUser() (Twilio SMS), and 
         /// startConversationWithUser() (Facebook Messenger) Be sure to check the platform documentation for others - most adapters include at least one.
         /// </summary>
         /// <param name="message">A string containing the text of a reply, or more fully formed message object</param>
         /// <returns>Return value will contain the results of the send action, typically {id: <id of message>}</returns>
-        public async Task<object> Say(IBotkitMessage message)
+        public async Task<ResourceResponse> Say(BotkitSlackMessage message)
         {
-            return new object();
+            var activity = EnsureMessageFormat(message);
+
+            return await config.TurnContext.SendActivityAsync(activity);
+        }
+
+        public async Task<ResourceResponse> Say(Activity message)
```

Comment:
I didn't see this overload in the Node code

Reply:
There isn't, however, `EnsureMessageFormat` returns an _Activity_, and say originally only accepted either `BotkitSlackMessage` or `string`.